packages/outputarea/src/model.ts

```diff
@@ -443,7 +442,7 @@ class OutputAreaModel implements IOutputAreaModel {
       this._changeGuard = false;
     }
     this._changed.emit(args);
-    this._stateChanged.emit(void 0);
+    this._stateChanged.emit(args.newValues[args.newIndex]);
```

Comment:
I don't think this is correct, as per the [interface](https://github.com/jupyterlab/jupyterlab/blob/master/packages/observables/src/observablelist.ts#L276). The new values are a contiguous list of new values, starting at the index `args.newIndex`. So a general change to the list of outputs has to allow for a list of outputs.

In practice, many outputs areas have just a single `IOutputModel`, and this works for that case, but I think it gives the wrong result for other cases.